third_party/terraform/tests/resource_google_service_account_test.go

```diff
@@ -18,7 +18,7 @@ func TestAccServiceAccount_basic(t *testing.T) {
 	displayName := "Terraform Test"
 	displayName2 := "Terraform Test Update"
 	desc := "test description"
-	desc2 := "test description update"
+	desc2 := ""
```

Comment:
This diff might also seem to be needless, but IMO setting empty value tend to be more buggy than setting other values, so I believe this is meaningful.